bindings/csharp/Ckzg.Test/ReferenceTests.cs

```diff
@@ -0,0 +1,337 @@
+using Microsoft.Extensions.FileSystemGlobbing;
+using NUnit.Framework;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Ckzg.Test;
+
+[TestFixture]
+public class ReferenceTests
+{
+    [OneTimeSetUp]
+    public void Setup()
+    {
+        _ts = Ckzg.LoadTrustedSetup("trusted_setup.txt");
+        _deserializer = new DeserializerBuilder().WithNamingConvention(new CamelCaseNamingConvention()).Build();
+    }
+
+    [OneTimeTearDown]
+    public void Teardown()
+    {
+        Ckzg.FreeTrustedSetup(_ts);
+    }
+
+    [TestCase]
+    public void TestSetupLoaded()
+    {
+        Assert.That(_ts, Is.Not.EqualTo(IntPtr.Zero));
+    }
+
+    private IntPtr _ts;
+    private const string TestDir = "../../../../../../tests";
+    private readonly string _blobToKzgCommitmentTests = Path.Join(TestDir, "blob_to_kzg_commitment");
+    private readonly string _computeKzgProofTests = Path.Join(TestDir, "compute_kzg_proof");
+    private readonly string _computeBlobKzgProofTests = Path.Join(TestDir, "compute_blob_kzg_proof");
+    private readonly string _verifyKzgProofTests = Path.Join(TestDir, "verify_kzg_proof");
+    private readonly string _verifyBlobKzgProofTests = Path.Join(TestDir, "verify_blob_kzg_proof");
+    private readonly string _verifyBlobKzgProofBatchTests = Path.Join(TestDir, "verify_blob_kzg_proof_batch");
```

Comment:
I believe the `TestDir` variable should be renamed:

```suggestion
    private const string _testDir = "../../../../../../tests";
    private readonly string _blobToKzgCommitmentTests = Path.Join(_testDir, "blob_to_kzg_commitment");
    private readonly string _computeKzgProofTests = Path.Join(_testDir, "compute_kzg_proof");
    private readonly string _computeBlobKzgProofTests = Path.Join(_testDir, "compute_blob_kzg_proof");
    private readonly string _verifyKzgProofTests = Path.Join(_testDir, "verify_kzg_proof");
    private readonly string _verifyBlobKzgProofTests = Path.Join(_testDir, "verify_blob_kzg_proof");
    private readonly string _verifyBlobKzgProofBatchTests = Path.Join(_testDir, "verify_blob_kzg_proof_batch");
```

Reply:
Also, should it be `readonly` too? What's the difference between `const` and `readonly`?

Reply:
Ah nevermind, I just [googled it](https://stackoverflow.com/a/56024) and it makes sense as a `const` value.

Reply:
`const` is for primitive values only, we can't init it with calls also. `readonly` covers the rest of the cases for immutable fields. 

Not very significant fact: `const` behaves more like C macros.

Reply:
The default convention is pascal case for constants. Looks a bit weird, but will show warnings in IDEs otherwise.

Reply:
Gotcha. Weird but I understand.